### PR TITLE
Docs: Adds a Glide snippet for the "Loading Images" Quick Guide at

### PR DESCRIPTION
**Pull request by thedmail**
https://developer.android.com/quick-guides/content/load-images?hl=en

### CODE REVIEW

File: compose/snippets/build.gradle.kts [hunk 1]
```diff
@@ -159,5 +159,6 @@ dependencies {
 
     androidTestImplementation(libs.androidx.compose.ui.test.junit4)
     debugImplementation(libs.androidx.compose.ui.test.manifest)
+    implementation("com.github.bumptech.glide:compose:1.0.0-beta01")
```
Comment (riggaroo) [inline compose/snippets/build.gradle.kts]: Please add the dependency to the libs.toml file and then reference the dependency like above libraries instead.

Comment (thedmail) [reply compose/snippets/build.gradle.kts]: Thanks. Will do.

Comment (thedmail) [reply compose/snippets/build.gradle.kts]: Tried to do it by imitating the way it's done for coil. Didn't work.

Is there documentation for what you're asking for?

Comment (thedmail) [reply compose/snippets/build.gradle.kts]: Think I got it figured out. It builds on my machine now. :-)

**Review by riggaroo — APPROVED** [review]

**Pull request merged**